README.md

```diff
@@ -17,26 +17,31 @@ pip install datascroller
 ```
 
 ## Usage
-Import the `scroll` function with
+### The binary
+In a command line environment where datascroller is installed, run
+`scroll <path to your csv>` and refer to the "Keys" Section for navigation.
```

Comment:
I think this would be helpful. The documentation for [Adding Non-Code Files](https://python-packaging.readthedocs.io/en/latest/non-code-files.html) discusses a way to do this that involves the MANIFEST.in file. The only other concern I had is what data could I put as part of the package without risking wrath of the data owner. I think these [Rdatasets](http://vincentarelbundock.github.io/Rdatasets/datasets.html) should be safe, but none I've found are as interesting as the Kaggle data sets. Ecdat/Car is ok. Have you started working on this yet? Otherwise I can take it today.

Reply:
Ok I'm catching up here! Nice work on setting up the demo data. Titanic is a safe choice. Wish it had a few more columns but it will do!

Reply:
<deleted a few comments I made while I was catching up>. Great idea to add Titanic! I thought it had like 6 columns and I was pleasantly surprised to see that it has 16. And it's a safe data set to include from a legal standpoint (even though a few court cases recently have ruled on the side of open data, why play with fire?).

The first tests of the binary got stuck finding the data. I saw you moved the demo_data folder into the module itself, and I'm currently continuing that strategy by creating a demo module that loads the data into memory. Then we shouldn't need the Manifest.in at all.

Reply:
16 Columns is plenty. I thought it had 6! Tried to delete my rambling above but Github isn't having it.